content/kubermatic/master/startio/guides/wizard/_index.en.md

```diff
@@ -0,0 +1,87 @@
++++
+title = "Use the Wizard to Configure KKP"
+weight = 10
+enableToc = true
++++
+
+Start by visiting [https://start.kubermatic.io/](https://start.kubermatic.io/).
+
+Introduction page will welcome you.
+
+Start by clicking on the **Generate** button.
+
+![Welcome Page](welcome.png?width=700px&classes=shadow,border "Welcome Page" )
+
+## 1. Provider selection
+At this step, you will select a cloud provider where the KKP will be deployed.
+
+![Step 1](1.png?width=700px&classes=shadow,border "Step 1")
+
+## 2. Kubernetes Cluster configuration
+At this step, you are providing details of your Kubernetes master cluster.
+
+You can generate a _Cluster name_, provide a Kubernetes _Master Cluster Version_.
+
+For the AWS setup, you can select an _AWS region_ where the cluster will be provisioned and also the _AWS Worker Type_
+(see the [Instance Types](https://aws.amazon.com/ec2/instance-types/), we recommend at least _t3.xlarge_ for the initial setup).
+{{% notice warning %}}
+Make sure to use the x86 instances, ARM instances are not supported.
+{{% /notice %}}
+
+Keep in mind that this configuration is for the master cluster where KKP will be deployed, later on you will provision
+User clusters through KKP where your workload will be deployed (and for that you may use a different instance types).
+
+![Step 2](2.png?width=700px&classes=shadow,border "Step 2")
+
+## 3. KKP configuration
+Here you are going to provide some high-level configuration of the KKP installation.
+
+_Version_ is matching the KKP release tag, see [Release page on github](https://github.com/kubermatic/kubermatic/releases).
+
+_Endpoint_ parameter represents the DNS endpoint where the KKP UI will be accessible in the browser (DNS registration will be described later).
+{{% notice warning %}}
+Don’t specify a protocol (https://) and the trailing slash in the _Endpoint_ input.
+{{% /notice %}}
+
+_Username_ should be your email which will be used for your initial user integrated inside Dex (used as a KKP authentication IdP),
+this user will be also “admin” of your KKP installation.
+
+There is optional choice of enabling the monitoring stack - if enabled, monitoring stack will be deployed on your kubernetes master cluster.
```

Comment:
Should this be optional, or do we recommend this?

Reply:
it's optional... and of course, we recommend it ;)